sql/hive/src/test/scala/org/apache/spark/sql/hive/ExpressionToSQLSuite.scala

```diff
@@ -0,0 +1,271 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.hive
+
+import scala.util.control.NonFatal
+
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.test.SQLTestUtils
+
+class ExpressionToSQLSuite extends SQLBuilderTest with SQLTestUtils {
```

Comment:
We will just move the entire suite over once we merge hive context and SQL
context.

On Tuesday, February 23, 2016, Sean Owen <notifications@github.com> wrote:

> In
> sql/hive/src/test/scala/org/apache/spark/sql/hive/ExpressionToSQLSuite.scala
> <https://github.com/apache/spark/pull/11314#discussion_r53756196>:
>
> > + *
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +
> > +package org.apache.spark.sql.hive
> > +
> > +import scala.util.control.NonFatal
> > +
> > +import org.apache.spark.sql.functions._
> > +import org.apache.spark.sql.test.SQLTestUtils
> > +
> > +class ExpressionToSQLSuite extends SQLBuilderTest with SQLTestUtils {
>
> Refactor the tests then? that much shouldn't dictate where you put
> non-test code
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/11314/files#r53756196>.
>